Header header
int16[] data # Represents raw wav audio data